### PR TITLE
Show extension dag command in `start/data-management/data-pipelines`

### LINKED ISSUE #4999: vs code: pipelines
### Report

VS Code can both help define the DAG and show the DAG. We need to show how this can help in https://dvc.org/doc/start/data-management/data-pipelines and/or https://dvc.org/doc/user-guide/pipelines.

### PR DESCRIPTION
**Pull request by julieg18**
## Demo

Screenshot 2024-01-22 at 6 47 10 PM

Fixes #4999

### CODE REVIEW

File: content/docs/start/data-management/data-pipelines.md [hunk 1]
```diff
@@ -400,6 +400,13 @@ Refer to `dvc dag` to explore other ways this command can visualize a pipeline.
 
 </admon>
 
+### Visualize in VS Code
+
+You can visualize the pipeline inside of VS Code with the
+[DVC extension for VS Code](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc):
+
+![Visualize the DAG in VS Code](/img/vscode-show-dag.gif)
+
```
Comment (julieg18) [inline content/docs/start/data-management/data-pipelines.md:409]: I chose a section with a gif, but open to other ideas/places!

**Comment by julieg18** [comment]
@dberenbaum, this pr has been sitting here for while, can you take a look when you have time?

**Review by dberenbaum — CHANGES_REQUESTED** [review]
Thanks for the ping @julieg18! Somehow this slipped by me.

What you added is fine, but I think it's more important to show how dvc.yaml support can help add and edit stages (using snippets, yaml schema hints, etc.). Do we have a video or gif for this? Could we show it above where we talk about adding stages?

**Comment by shcheklein** [comment]
Closing it for now, let's get back to it later, when we have better idea about the get started structure.

**Pull request closed without merging**